controllers/utils/utils.go

```diff
@@ -17,7 +17,10 @@ limitations under the License.
 package utils
 
 import (
+	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"reflect"
+	"time"
```

Comment:
can you organize the imports to the the core ones will be above and after new line the pulled ones will be after it, like [here](https://blog.devgenius.io/sort-go-imports-acb76224dfa7)

Reply:
done